src/Symfony/Component/Form/Tests/AbstractBootstrap3LayoutTest.php

```diff
@@ -968,8 +960,7 @@ public function testMultipleChoiceExpandedAttributes()
         $form = $this->factory->createNamed('name', 'Symfony\Component\Form\Extension\Core\Type\ChoiceType', array('&a', '&c'), array(
             'choices' => array('Choice&A' => '&a', 'Choice&B' => '&b', 'Choice&C' => '&c'),
             'choice_attr' => array('Choice&B' => array('class' => 'foo&bar')),
-            'multiple' => true,
-            'expanded' => true,
+            'widget' => 'checkbox',
             'required' => true,
```

Comment:
Same for legacy test with `MultipleChoicesExpanded` without `widget` option and with `expanded` and `multiple` set to true.